compiler/luci/import/src/Importer.cpp

```diff
@@ -122,12 +122,19 @@ void convert_graph(const luci::GraphBuilderSource &source, luci::CircleReader &r
   }
 
   // Create CircleConst nodes for constant tensors.
+  // Create CircleNodes for constant tensors.
   // NOTE Origin is intentionally not provided for constants.
+  auto const_builder = source.lookup(luci::NodeBuilderType::BUFFER);
```

Comment:
Can you please point where builder is registered with `BUFFER`?

Reply:
Builder is registred in same way as builder regiters for other nodes:

https://github.com/Samsung/ONE/blob/daea9fdc9f8a6338d14d2be88c14f7187256be93/compiler/luci/import/src/GraphBuilderRegistry.cpp#L165-L170